lib/fog/vsphere/requests/compute/list_child_snapshots.rb

```diff
@@ -0,0 +1,68 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def list_child_snapshots(snapshot, opts = {})
+          s = Hash === snapshot ?
+            Snapshot.new(snapshot.update(service: self)) : snapshot
+          s.tree_node.childSnapshotList.map do |snap|
```

Comment:
Damm. Is not in your hands then. Too bad the guys on `rbvmomi` made such a confusion.
The mixed up both snake_case and camelCase. =s

From their own example:

```ruby
vim.serviceInstance.find_datacenter("mydatacenter")
```

=s